bats/tests/containers/wasm.bats

```diff
@@ -0,0 +1,149 @@
+# shellcheck disable=SC2030,SC2031
+# See https://github.com/koalaman/shellcheck/issues/2431
+# https://www.shellcheck.net/wiki/SC2030 -- Modification of output is local (to subshell caused by @bats test)
+# https://www.shellcheck.net/wiki/SC2031 -- output was modified in a subshell. That change might be lost
+
+load '../helpers/load'
+
+# Bundled shims are this version or newer.
+BUNDLED_VERSION=0.11.1
+# Manually managed versions intentionally use an older version
+# so we can verify that they still override the bundled version.
+MANUAL_VERSION=0.10.0
+
+local_setup() {
+    if using_containerd; then
+        skip "this test only works on moby right now"
+    fi
+}
+
+@test 'factory reset' {
+    factory_reset
+    rm -rf "$PATH_CONTAINERD_SHIMS"
```

Comment:
It would be good to have a `local_teardown` that also removes any shims, to avoid affecting other tests.

Reply:
As I wrote above, other tests are expected to clean up at the start, but I don't want to leave older shims around for the user, so added the teardown.